api/migrations/028_add_ldapuser_column_in_user.js

```diff
@@ -24,7 +24,7 @@
 
 function up(knex) {
   return knex.schema.table('user', (table) => {
-    table.string('ldapUser');
+    table.boolean('ldapUser');
   });
 }
 
```

Comment:
This migration number 028, is already used.
https://github.com/Nanocloud/nanocloud/blob/master/api/migrations/028_add_appName_on_history.js